app/src/main/res/layout/row_course_module_resource.xml

```diff
@@ -1,132 +1,132 @@
-            android:autoLink="all"
-            android:clickable="true"
-            android:focusable="true"
-            android:paddingLeft="8dp"
-            android:paddingRight="8dp"
-            android:textColor="?android:textColorSecondary"
-            android:text="@string/lorem" />
-
-        <View
-            android:id="@+id/bottomDivider"
-            android:layout_width="match_parent"
-            android:layout_height="1dp"
-            android:background="@android:color/darker_gray" />
-    </LinearLayout>
-</RelativeLayout>
-
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:background="?themeBackground"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:clipChildren="false">
+
+    <RelativeLayout
+        android:id="@+id/iconWrapper"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingBottom="8dp"
+        android:paddingLeft="8dp"
+        android:paddingRight="8dp"
+        android:paddingTop="0dp">
+
+        <ImageView
+            android:id="@+id/fileIcon"
+            android:layout_width="48dp"
+            android:layout_height="48dp"
+            android:layout_centerVertical="true"
+            android:padding="8dp"
+            app:srcCompat="@drawable/file_powerpoint" />
+
+        <ProgressBar
+            android:id="@+id/progressBar"
+            android:layout_width="47dp"
+            android:layout_height="43dp"
+            android:layout_centerVertical="true"
+            android:padding="12dp" />
+    </RelativeLayout>
+
+    <LinearLayout
+        android:id="@+id/textWrapper"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_toRightOf="@+id/iconWrapper"
+        android:clipChildren="false"
+        android:orientation="vertical">
+
+        <View
+            android:id="@+id/topDivider"
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="@android:color/darker_gray" />
+
+        <LinearLayout
+            android:id="@+id/clickWrapper"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:clipChildren="false"
+            android:orientation="horizontal"
+            android:paddingBottom="6dp"
+            android:paddingTop="6dp">
+
+            <LinearLayout
+                android:id="@+id/clickWrapperName"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:clickable="true"
+                android:focusable="true"
+                android:background="?android:selectableItemBackground"
```

Comment:
Except for this line, why have other lines been changed? I think you have changed the indentation type from tab to space or vice-versa. Please restore it as it was and only change the required lines.